src/GovUk.Education.ExploreEducationStatistics.Admin/Mappings/MappingProfiles.cs

```diff
@@ -222,6 +224,46 @@ public MappingProfiles()
                 .ForMember(dest => dest.Type,
                     m => m.MapFrom(dataSetVersion =>
                         dataSetVersion.VersionType));
+
+            CreateMap<DataSetUpload, DataSetUploadViewModel>()
```

Comment:
Nice :sunglasses:  I'm still on the fence with AutoMapper, but this is a good use of its customization!